tests/__tests__/options.test.ts

```diff
@@ -831,29 +1019,11 @@ describe('Middleware options', () => {
         const { _handleSet } =
           store.temporal.getState() as _TemporalState<MyState>;
         act(() => {
-          _handleSet(store.getState());
+          _handleSet(store.getState(), undefined, store.getState(), null);
         });
         expect(store.temporal.getState().pastStates.length).toBe(1);
       });
 
-      it('should only update if the the status is tracking', () => {
```

Comment:
Removed this test, as conditional history tracking depending on `isTracking` is no longer handled by the internal `_handleSet` function. Instead it is now handled by `temporalHandleSet` in `index.ts`. Tracking functionality is still successfully tested in other tests